Traceback (most recent call last):
  File "/usr/share/miniconda/lib/python3.11/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/usr/share/miniconda/lib/python3.11/site-packages/nbclient/client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/share/miniconda/lib/python3.11/site-packages/jupyter_core/utils/__init__.py", line 166, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/share/miniconda/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/usr/share/miniconda/lib/python3.11/site-packages/nbclient/client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "/usr/share/miniconda/lib/python3.11/site-packages/nbclient/client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/usr/share/miniconda/lib/python3.11/site-packages/nbclient/client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
acs_destripe_plus.destripe_plus(sbc_fits, cte_correct=True)
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mFileNotFoundError[0m                         Traceback (most recent call last)
Cell [0;32mIn[11], line 1[0m
[0;32m----> 1[0m [43macs_destripe_plus[49m[38;5;241;43m.[39;49m[43mdestripe_plus[49m[43m([49m[43msbc_fits[49m[43m,[49m[43m [49m[43mcte_correct[49m[38;5;241;43m=[39;49m[38;5;28;43;01mTrue[39;49;00m[43m)[49m

File [0;32m/usr/share/miniconda/lib/python3.11/site-packages/acstools/acs_destripe_plus.py:373[0m, in [0;36mdestripe_plus[0;34m(inputfile, suffix, stat, maxiter, sigrej, lower, upper, binwidth, scimask1, scimask2, dqbits, rpt_clean, atol, cte_correct, keep_intermediate_files, clobber, verbose)[0m
[1;32m    367[0m [38;5;28;01mif[39;00m (header[[38;5;124m'[39m[38;5;124mCCDAMP[39m[38;5;124m'[39m] [38;5;241m==[39m [38;5;124m'[39m[38;5;124mABCD[39m[38;5;124m'[39m [38;5;129;01mand[39;00m
[1;32m    368[0m         ((scimask1 [38;5;129;01mis[39;00m [38;5;129;01mnot[39;00m [38;5;28;01mNone[39;00m [38;5;129;01mand[39;00m scimask2 [38;5;129;01mis[39;00m [38;5;28;01mNone[39;00m) [38;5;129;01mor[39;00m
[1;32m    369[0m          (scimask1 [38;5;129;01mis[39;00m [38;5;28;01mNone[39;00m [38;5;129;01mand[39;00m scimask2 [38;5;129;01mis[39;00m [38;5;129;01mnot[39;00m [38;5;28;01mNone[39;00m))):
[1;32m    370[0m     [38;5;28;01mraise[39;00m [38;5;167;01mValueError[39;00m([38;5;124m"[39m[38;5;124mBoth [39m[38;5;124m'[39m[38;5;124mscimask1[39m[38;5;124m'[39m[38;5;124m and [39m[38;5;124m'[39m[38;5;124mscimask2[39m[38;5;124m'[39m[38;5;124m must be specified [39m[38;5;124m"[39m
[1;32m    371[0m                      [38;5;124m"[39m[38;5;124mor not specified together.[39m[38;5;124m"[39m)
[0;32m--> 373[0m calacs_str [38;5;241m=[39m [43msubprocess[49m[38;5;241;43m.[39;49m[43mcheck_output[49m[43m([49m[43m[[49m[38;5;124;43m'[39;49m[38;5;124;43mcalacs.e[39;49m[38;5;124;43m'[39;49m[43m,[49m[43m [49m[38;5;124;43m'[39;49m[38;5;124;43m--version[39;49m[38;5;124;43m'[39;49m[43m][49m[43m)[49m[38;5;241m.[39msplit()[[38;5;241m0[39m]  [38;5;66;03m# nosec # noqa[39;00m
[1;32m    374[0m calacs_ver [38;5;241m=[39m [[38;5;28mint[39m(x) [38;5;28;01mfor[39;00m x [38;5;129;01min[39;00m calacs_str[38;5;241m.[39mdecode()[38;5;241m.[39msplit([38;5;124m'[39m[38;5;124m.[39m[38;5;124m'[39m)]
[1;32m    375[0m [38;5;28;01mif[39;00m calacs_ver [38;5;241m<[39m [[38;5;241m8[39m, [38;5;241m3[39m, [38;5;241m1[39m]:

File [0;32m/usr/share/miniconda/lib/python3.11/subprocess.py:466[0m, in [0;36mcheck_output[0;34m(timeout, *popenargs, **kwargs)[0m
[1;32m    463[0m         empty [38;5;241m=[39m [38;5;124mb[39m[38;5;124m'[39m[38;5;124m'[39m
[1;32m    464[0m     kwargs[[38;5;124m'[39m[38;5;124minput[39m[38;5;124m'[39m] [38;5;241m=[39m empty
[0;32m--> 466[0m [38;5;28;01mreturn[39;00m [43mrun[49m[43m([49m[38;5;241;43m*[39;49m[43mpopenargs[49m[43m,[49m[43m [49m[43mstdout[49m[38;5;241;43m=[39;49m[43mPIPE[49m[43m,[49m[43m [49m[43mtimeout[49m[38;5;241;43m=[39;49m[43mtimeout[49m[43m,[49m[43m [49m[43mcheck[49m[38;5;241;43m=[39;49m[38;5;28;43;01mTrue[39;49;00m[43m,[49m
[1;32m    467[0m [43m           [49m[38;5;241;43m*[39;49m[38;5;241;43m*[39;49m[43mkwargs[49m[43m)[49m[38;5;241m.[39mstdout

File [0;32m/usr/share/miniconda/lib/python3.11/subprocess.py:548[0m, in [0;36mrun[0;34m(input, capture_output, timeout, check, *popenargs, **kwargs)[0m
[1;32m    545[0m     kwargs[[38;5;124m'[39m[38;5;124mstdout[39m[38;5;124m'[39m] [38;5;241m=[39m PIPE
[1;32m    546[0m     kwargs[[38;5;124m'[39m[38;5;124mstderr[39m[38;5;124m'[39m] [38;5;241m=[39m PIPE
[0;32m--> 548[0m [38;5;28;01mwith[39;00m [43mPopen[49m[43m([49m[38;5;241;43m*[39;49m[43mpopenargs[49m[43m,[49m[43m [49m[38;5;241;43m*[39;49m[38;5;241;43m*[39;49m[43mkwargs[49m[43m)[49m [38;5;28;01mas[39;00m process:
[1;32m    549[0m     [38;5;28;01mtry[39;00m:
[1;32m    550[0m         stdout, stderr [38;5;241m=[39m process[38;5;241m.[39mcommunicate([38;5;28minput[39m, timeout[38;5;241m=[39mtimeout)

File [0;32m/usr/share/miniconda/lib/python3.11/subprocess.py:1026[0m, in [0;36mPopen.__init__[0;34m(self, args, bufsize, executable, stdin, stdout, stderr, preexec_fn, close_fds, shell, cwd, env, universal_newlines, startupinfo, creationflags, restore_signals, start_new_session, pass_fds, user, group, extra_groups, encoding, errors, text, umask, pipesize, process_group)[0m
[1;32m   1022[0m         [38;5;28;01mif[39;00m [38;5;28mself[39m[38;5;241m.[39mtext_mode:
[1;32m   1023[0m             [38;5;28mself[39m[38;5;241m.[39mstderr [38;5;241m=[39m io[38;5;241m.[39mTextIOWrapper([38;5;28mself[39m[38;5;241m.[39mstderr,
[1;32m   1024[0m                     encoding[38;5;241m=[39mencoding, errors[38;5;241m=[39merrors)
[0;32m-> 1026[0m     [38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43m_execute_child[49m[43m([49m[43margs[49m[43m,[49m[43m [49m[43mexecutable[49m[43m,[49m[43m [49m[43mpreexec_fn[49m[43m,[49m[43m [49m[43mclose_fds[49m[43m,[49m
[1;32m   1027[0m [43m                        [49m[43mpass_fds[49m[43m,[49m[43m [49m[43mcwd[49m[43m,[49m[43m [49m[43menv[49m[43m,[49m
[1;32m   1028[0m [43m                        [49m[43mstartupinfo[49m[43m,[49m[43m [49m[43mcreationflags[49m[43m,[49m[43m [49m[43mshell[49m[43m,[49m
[1;32m   1029[0m [43m                        [49m[43mp2cread[49m[43m,[49m[43m [49m[43mp2cwrite[49m[43m,[49m
[1;32m   1030[0m [43m                        [49m[43mc2pread[49m[43m,[49m[43m [49m[43mc2pwrite[49m[43m,[49m
[1;32m   1031[0m [43m                        [49m[43merrread[49m[43m,[49m[43m [49m[43merrwrite[49m[43m,[49m
[1;32m   1032[0m [43m                        [49m[43mrestore_signals[49m[43m,[49m
[1;32m   1033[0m [43m                        [49m[43mgid[49m[43m,[49m[43m [49m[43mgids[49m[43m,[49m[43m [49m[43muid[49m[43m,[49m[43m [49m[43mumask[49m[43m,[49m
[1;32m   1034[0m [43m                        [49m[43mstart_new_session[49m[43m,[49m[43m [49m[43mprocess_group[49m[43m)[49m
[1;32m   1035[0m [38;5;28;01mexcept[39;00m:
[1;32m   1036[0m     [38;5;66;03m# Cleanup if the child failed starting.[39;00m
[1;32m   1037[0m     [38;5;28;01mfor[39;00m f [38;5;129;01min[39;00m [38;5;28mfilter[39m([38;5;28;01mNone[39;00m, ([38;5;28mself[39m[38;5;241m.[39mstdin, [38;5;28mself[39m[38;5;241m.[39mstdout, [38;5;28mself[39m[38;5;241m.[39mstderr)):

File [0;32m/usr/share/miniconda/lib/python3.11/subprocess.py:1950[0m, in [0;36mPopen._execute_child[0;34m(self, args, executable, preexec_fn, close_fds, pass_fds, cwd, env, startupinfo, creationflags, shell, p2cread, p2cwrite, c2pread, c2pwrite, errread, errwrite, restore_signals, gid, gids, uid, umask, start_new_session, process_group)[0m
[1;32m   1948[0m     [38;5;28;01mif[39;00m errno_num [38;5;241m!=[39m [38;5;241m0[39m:
[1;32m   1949[0m         err_msg [38;5;241m=[39m os[38;5;241m.[39mstrerror(errno_num)
[0;32m-> 1950[0m     [38;5;28;01mraise[39;00m child_exception_type(errno_num, err_msg, err_filename)
[1;32m   1951[0m [38;5;28;01mraise[39;00m child_exception_type(err_msg)

[0;31mFileNotFoundError[0m: [Errno 2] No such file or directory: 'calacs.e'

